crates/codegen/language/definition/src/internals/parse_input_tokens/helpers.rs

```diff
@@ -57,7 +57,7 @@ impl ParseHelpers {
     pub fn map<K: ParseInputTokens + std::hash::Hash + Eq, V: ParseInputTokens>(
         input: ParseStream,
         errors: &mut ErrorsCollection,
-    ) -> Result<IndexMap<Spanned<K>, Spanned<V>>> {
+    ) -> Result<IndexMap<Spanned<K>, V>> {
```

Comment:
Do we always use Spanned<Identifier> as the key here?

Reply:
So far yes, but there is no need for the restriction. Will remove it.